framework/src/outputs/JsonIO.C

```diff
@@ -44,7 +44,7 @@ to_json(nlohmann::json & json, const MooseApp & app)
 namespace libMesh
 {
 void
-to_json(nlohmann::json & json, const Point & p)
+to_json(nlohmann::json & json, const libMesh::Point & p)
```

Comment:
With the declaration and definition now in the `libMesh` namespace this isn't necessary anymore. (But it's also not a problem either)